view/theme/frio/templates/settings_features.tpl

```diff
@@ -19,7 +19,11 @@
 				<div id="{{$g}}-settings-content" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{$g}}-settings-title">
 					<div class="section-content-tools-wrapper">
 						{{foreach $f.1 as $fcat}}
+							{{* Here is an addional condition to filter the richtext feature
+							because frio doesn't support the richtext *}}
+							{{if $fcat.0 != 'feature_richtext'}}
```

Comment:
You can add translatable strings to template with something like
```
{{"Text"|t}}
```
(which is a Smarty3 feature: t() is defined globally, and that tag will be evaluated as `t("Text")`.)
but the run_xgettext script doesn't extract them yet.